bin/sum

```diff
@@ -195,12 +207,11 @@ sum - display file checksums and block counts
 
 B<sum>
 [ B<-o> I<0|1|2> ]
-[ B<-h> ]
 [ I<filename ...> ]
 
 =head1 DESCRIPTION
 
-sum outputs three space seperated values:  file CRC, file size, and
+sum outputs three space separated values:  file CRC, file size, and
```

Comment:
This must be the most common typo. You've fixed this in more than a couple programs I think.